BUILDGUIDE.md

```diff
@@ -242,6 +242,12 @@ To use this feature, you must enable the following AppContext switch at applicat
 
 **"Switch.Microsoft.Data.SqlClient.EnableRetryLogic"**
 
+## Enabling row version null behavior
+
+`SqlDataReader` returns a `DBNull` value instead of an empty `byte[]`. To active the legacy behavior, you must enable the following AppContext switch that is cached on first check for the entire process:
```

Comment:
```suggestion
`SqlDataReader` returns a `DBNull` value instead of an empty `byte[]`. To enable the legacy behavior, you must enable the following AppContext switch on application startup:
```